CitesphereConnector.py

```diff
@@ -0,0 +1,106 @@
+
+import urllib2
+import json
+
+# CITESPHERE_API = "https://citesphere.instance.org/api/v1"
+CITESPHERE_API = "https://diging-dev.asu.edu/citesphere-review/api"
```

Comment:
This should be a configurable value. Someone who uses the library might connect to any instance, not necessarily always the same.